mobile portfolio landing page

tjay1760 opened the pull request:
In this pull request I have completed the following tasks:
- Created an index.html file that has various sections that give a brief personal introduction
- I created a style.css file that has responsive styling for mobile devices with a screen size of less than 450px 
- I added linters to the project to help detect any styling errors
- Updated my README.md file

KanzaTahreem reviewed (commented):
# INVALID ❌ 

Hi @tjay1760 👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😓

## Invalid Code Review Request ❌
- [ ] Unfortunately, you have submitted a project that is not professional and lacks a descriptive README file 📖. Please make sure that you add a professional README file. You can follow the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) and  make sure that all headers are present, but the content should be customized to this project repository.
- [ ] Also please make sure to add a descriptive title to Pull request telling you what changes are inside and a descriptive short summary describing introduced changes in more detail.

Please ensure that you professionally document the README file to reflect the project

You can take a step back and read the [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for more clarity 💡

_**Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready**_ 🤝🙏

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

_**Reviewed By [Kanza Tahreem](https://github.com/KanzaTahreem)**_

tjay1760 commented:
In this pull request : 
I have updated the README.md file
I have created an index.html file for the landing page
I added a styles.css file to style the content 
I have added a flexbox with sections as flex items

osamaashraf6 reviewed (commented):
Hi @tjay1760 😊, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🆘💀

I see 😃 that you submitted the pull request PR for multiple milestones and you are now at the finished mobile version You have submitted a project that :

- [ ] the PR title is not having a meaningful and descriptive name;
- [ ] important headers in the README are without content;
- [ ] deployment header should be removed for now;💯
- [ ] and kindly do not use the same PR for multiple milestones

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

mckent05 reviewed (commented):
Hi @tjay1760, 

# STATUS: INVALID REQUEST ❌ ❌ 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that is invalid for the following reasons:
- Your PR title and summary are not descriptive, your PR title should be a summary of the changes made in the PR, while your PR summary should describe in little more detail the changes made in the PR. Please fix this 🙏 
To edit your PR title, open your PR and click on the 'edit' button, reference the picture attached below:
<img width="941" alt="rgwrgwg" src="https://user-images.githubusercontent.com/73607512/204491103-03d8df16-2137-4c39-94e7-f93197a613b3.PNG">

To edit your PR sumary, open your PR and click on the button, reference the picture attached below:
<img width="620" alt="fwhaeth" src="https://user-images.githubusercontent.com/73607512/204491337-71638dde-7add-4d5e-aee9-880c83d55b2a.PNG">
- Please do not use the same PR for the same milestone 
- Also, please ensure that all headers in your readme have a corresponding section

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

huseyinbiyik commented on index.html:
```diff
@@ -0,0 +1,40 @@
+            <img src="/media/My Logo.png">
+        </div>
+        <nav>
+            <div class="line"></div>
+            <div class="line"></div>
+            <div class="line"></div>
```
Please make sure your color matches for the hamburger menu. Check the color difference below.
<img width="172" alt="Screenshot 2023-01-18 at 11 27 17 PM" src="https://user-images.githubusercontent.com/25887253/213287651-137e2e60-3e2e-4041-a03d-96fd985b2e7e.png">
<img width="172" alt="Screenshot 2023-01-18 at 11 27 34 PM" src="https://user-images.githubusercontent.com/25887253/213287656-0b79e07c-4821-441a-a8a8-cce741d7f809.png">

IngDavidTM commented on index.html:
```diff
@@ -0,0 +1,64 @@
+        </nav>
+    </header>
+    <section id="greeting">
+        <p>I'm John Glad to see you</p>
```
- [ ] As required before you should use a `<h1>` element here instead of using `<p>` because it will help search engines understand the meaning (context) of a page :+1:. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info

topeogunleye replied:
- [ ] As required by the previous reviewer, kindly ensure that there should be an `h1` in an HTML file because of semantics and it helps search engines and web crawlers easily understand the main subject of your page. I kindly suggest that you make only your main title here an `h1`

IngDavidTM commented on index.html:
```diff
@@ -0,0 +1,64 @@
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+    </section>
+    <section id="socials">
+        <h2 id="connect">LET'S CONNECT</h2>
+        <div id="social-icons"> 
+            <img src="/media/social media icons.png">
```
- [ ] As mentioned before please instead of using a `<img>` tag for the social icons, as a project requirement, you should create each of them. Please use `<ul>` tag to group them, and `<li>` per icon; each icon should have a link (`<a>` tag). Because apart from being semantically correct, the `<a>` tag will let you connect your social networks with the icons, using `href` :+1:

topeogunleye replied:
- [ ]  As mentioned by the two previous code reviewers before,  kindly update your code to make it more semantic and for the best HTML practices : the social icons should be listed inside a list `<ul> & <li>` instead of normal divs, and each social icon should be an `<a>` that contains the link for that icon
>   example:
> 
> ```
> <ul class='...'>
> <li class='...'><a href='...' ><svg>...</svg></a></li>
> <li class='...'><a href='...' ><svg>...</svg></a></li>
> ...
> <li class='...'><a href='...' ><svg>...</svg></a></li>
> </ul>
> ```

IngDavidTM commented on index.html:
```diff
@@ -0,0 +1,64 @@
+            <img src="/media/social media icons.png">
+        </div>
+    </section>
+    <!--<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Projects</title>
+</head>
+<body>
+<div id ="project">
+    <img src="media/Snapshoot Portfolio.png">
+</div>
+<div id ="project-title">
+    Tonic
+</div>
+<div id="project-description"></div>
+<div id="project-detail"></div>
+<div id="languages-used"></div>
+<div id="view-project"></div>
+
+</body>
+</html>
+-->
```
- [ ] As a general requirement you shouldn't commit commented-out code. It distracts the programmer who is reading your code and tempts them to try to figure out why the code was commented on and what it would do if it weren't. Additionally, it doesn't make the code looks neat :+1:. Here you can find more information about that ["The danger of commenting out code"](https://www.markhneedham.com/blog/2009/01/17/the-danger-of-commenting-out-code/)

IngDavidTM reviewed (changes requested):
## Hi @tjay1760  🙋‍♂️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

<img src="https://media2.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif"/>

## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

topeogunleye commented on index.html:
```diff
@@ -0,0 +1,64 @@
+</head>
+<body>
+    <div id ="content">
+    <div id ="status-bar"></div>
```
- [ ] As suggested by the previous reviewer, I kindly suggest that you remove the are showing the time, battery icon signal icon and wifi icon.

<img width="318" alt="image" src="https://user-images.githubusercontent.com/52934128/213402489-8e0e6b3e-27dc-4e84-ba42-c33a56d19e08.png">

topeogunleye commented on index.html:
```diff
@@ -0,0 +1,64 @@
+            <div class="line"></div>
+        </nav>
+    </header>
+    <section id="greeting">
```
- [ ] As required by the previous reviewer, this is the only place where you need a section tag, please replace every other section tag with a div or an appropriate tag for it

topeogunleye commented on index.html:
```diff
@@ -0,0 +1,64 @@
+            <img src="/media/social media icons.png">
+        </div>
+    </section>
+    <!--<!DOCTYPE html>
```
- [ ] As suggested by the previous reviewer, please remove commented out code

tjay1760 replied:
Hello can you check again @topeogunleye It appears the push did not go through when I sent it. Sorry for the inconvinence caused.

topeogunleye replied:
@tjay1760, I'll check and get back to you

topeogunleye replied:
@tjay1760, The change is there now, the commented out code is gone 👏

topeogunleye reviewed (changes requested):
Hi @tjay1760, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* Good commit messages. :heavy_check_mark:
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:
* Professional README file. :heavy_check_mark:
* PR has a good title and summary :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

GedeonTS reviewed (changes requested):
# INVALID REVIEW REQUEST

Hi @tjay1760 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
<Img src="https://media.giphy.com/media/11tVh1XRNAunYI/giphy.gif">
## Invalid Code Review Request

You have submitted a project, that is not for this milestone, kindly submit it to the right one through your dashboard, you can request an extra review if you have used all your three review requests

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

RiyaBulia12 commented on index.html:
```diff
@@ -0,0 +1,42 @@
+    <section class ="content">
+    <header>
+        <a id="logo">
+            <img src="/media/My Logo.png">
```
- **[OPTIONAL]** It looks like the image URL is broken as it is not being displayed. I highly recommend fixing this issue will improve your UI. 💯 
- **[OPTIONAL]** The hamburger menu is not aligned properly and does not match the Figma template. I would also recommend fixing this issue. 😄 

YOUR DESIGN
<img width="374" alt="Screenshot 2023-01-19 at 4 34 43 PM" src="https://user-images.githubusercontent.com/24832846/213426361-1bb0fc04-33c9-4fb8-b238-c4d97f6c262e.png">

FIGMA DESIGN
<img width="431" alt="Screenshot 2023-01-19 at 4 36 17 PM" src="https://user-images.githubusercontent.com/24832846/213426669-1e64d70d-8b71-4263-846b-0b5f6afc8607.png">

RiyaBulia12 commented on styles.css:
```diff
@@ -0,0 +1,88 @@
+
+@media (max-width: 500px) {
+  body {
+    background-image: url("media/header-shapes mobile.png");
```
- **[OPTIONAL]** Great job on applying the background image through CSS. 🥇  However, your background image is repeating which does not match the Figma template. It would be nice to fix the background image as it appeals to your design. 💯 😃 
 
<img width="376" alt="Screenshot 2023-01-19 at 4 38 34 PM" src="https://user-images.githubusercontent.com/24832846/213427144-99a4c517-6707-4182-8406-27554d7df45b.png">

Figma Design
<img width="350" alt="Screenshot 2023-01-19 at 4 39 50 PM" src="https://user-images.githubusercontent.com/24832846/213427410-f3809e50-70b6-419c-9a59-f065c47b6259.png">

RiyaBulia12 commented on index.html:
```diff
@@ -0,0 +1,42 @@
+            <div class="line"></div>
+        </nav>
+    </header>
+    <div id="greeting">
```
- **[OPTIONAL]** This section seems too close to the navbar. Adding more spacing to it will help the Intro page to fill the remaining screen space. It would be great if you could add space to match it with the Figma template. 👍🏽 

<img width="371" alt="Screenshot 2023-01-19 at 4 43 10 PM" src="https://user-images.githubusercontent.com/24832846/213428114-ef647040-897b-4352-8210-4981623cc618.png">

Figma Template
<img width="353" alt="Screenshot 2023-01-19 at 4 43 25 PM" src="https://user-images.githubusercontent.com/24832846/213428181-aa05f3cd-c378-472d-9347-c2d5b0580a4c.png">

RiyaBulia12 reviewed (approved):
## STATUS: APPROVED 🏆🥇 

### Hi @tjay1760 👋🏽  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
#### Congratulations! 🎉 🎉 🥳 🥳 

<img src="https://user-images.githubusercontent.com/50941074/190630799-31045ee0-2625-4964-92fa-9e77f79f6fda.gif"/>

## Highlights
* No Linter errors ✔️
* Professional Readme file ✔️
* Addressed all the previous comments ✔️ 
* GitHub flow used ✔️
* Deployed project ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
